tests/scripts/all.sh

```diff
@@ -832,9 +832,21 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
     if_build_succeeded env OPENSSL_CMD="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
 }
```

Comment:
In the commit message of commit "Add all.sh component that exercises invalid_param checks ", this sentence doesn't make sense.

"Add a component to all.sh that enables MBEDTLS_CHECK_PARAMS but disables MBEDTLS_CHECK_PARAMS_ASSERT doesn't define MBEDTLS_PARAM_FAILED so that these tests are exercised."

Should this be like so? (I only added an "and".)

"Add a component to all.sh that enables MBEDTLS_CHECK_PARAMS but disables MBEDTLS_CHECK_PARAMS_ASSERT and doesn't define MBEDTLS_PARAM_FAILED so that these tests are exercised."

Reply:
I fixed this and reworded the commit message to make it easier to read.